8.2.6, v1 recipe

minrk opened the pull request:
includes native mac arm builds, so we don't need to deal with cross anymore:

closes #64 
closes #50 
closes #30

minrk committed:
v1 recipe, 8.2.6

minrk committed:
MNT: Re-rendered with conda-build 25.3.1, conda-smithy 3.48.1, and conda-forge-pinning 2025.05.06.07.43.36

minrk committed:
avoid issues with cmake 4 removals

minrk committed:
need xorgproto for X11/Xosdefs.h

minrk committed:
remove libcxx pin on mac

minrk committed:
MNT: Re-rendered with conda-build 25.4.2, conda-smithy 3.48.1, and conda-forge-pinning 2025.05.06.07.43.36

minrk committed:
pip, verbose build

minrk committed:
constrain numpy<2

minrk committed:
MNT: Re-rendered with conda-build 25.4.2, conda-smithy 3.48.1, and conda-forge-pinning 2025.05.06.07.43.36

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

minrk committed:
Merge remote-tracking branch 'origin/main' into 826

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

minrk committed:
preserve cmake_args

not sure why mac isn't properly including x11

minrk committed:
don't build interviews on mac

x11 is weird and unreliable on mac

minrk committed:
rebuild for mpich 5

minrk committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.05.12.12.28.02

Other tools:
- conda-build 26.3.0
- rattler-build 0.64.1
- rattler-build-conda-compat 1.4.14

minrk committed:
native mac arm builds

minrk committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.05.12.12.28.02

Other tools:
- conda-build 26.3.0
- rattler-build 0.64.1
- rattler-build-conda-compat 1.4.14

minrk committed:
remove linux-only deps from run

these are in run_exports already

minrk committed:
Sync with main

minrk committed:
test: fix arch path

minrk committed:
skip 3.13 for now

minrk committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.05.12.12.28.02

Other tools:
- conda-build 26.3.0
- rattler-build 0.64.1
- rattler-build-conda-compat 1.4.14

minrk committed:
arch is wrong on osx-64

minrk committed:
make sure target_platform is set
